DuckDuckGo/TabViewController.swift

```diff
@@ -184,7 +190,14 @@ class TabViewController: WebViewController {
     
     private func refreshAction() -> UIAlertAction {
         return UIAlertAction(title: UserText.actionRefresh, style: .default) { [weak self] action in
-            self?.reload()
+            guard let sself = self else { return }
```

Comment:
Optional: Call this `strongSelf` that's the pattern I've seen in Apple's code. I used to write `weakSelf` because it was a ref to [weak self] but I don't like that anymore as the reference becomes a strong one so it feels disingenuous!